src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
Consider adding the `crossorigin` attribute to the `preconnect` link for `fonts.gstatic.com` to improve performance. This is recommended when using `preconnect` for external resources that require CORS.